getting-started-async/src/main/java/org/acme/quickstart/async/GreetingResource.java

```diff
@@ -1,4 +1,4 @@
-package org.acme.quickstart;
+package org.acme.quickstart.async;
```

Comment:
Some of them have `quickstart` in it but not all of them? Should we make everything consistent? @rsvoboda WDYT?

BTW, we will need to upgrade the guides too.

Reply:
+1 to have it consistent

I asked @Rutzzzz to prepare list of proposed package names.

He should paste the list here so we can agree on it before he starts adjusting the code.